src/generator/endpoints/elevenLabsApi.py

```diff
@@ -0,0 +1,102 @@
+import json
+import logging
+from datetime import datetime
+
+from fastapi import APIRouter, FastAPI, HTTPException, UploadFile, File
+from pydantic import BaseModel
+import elevenLabsLogic
+import os
+
+API_KEY = "a8412809ee48754b006d19396dd7eb40"
+
+# API_KEY = os.getenv("API_KEY")
+# if API_KEY == "":
+#     raise Exception("ElevenLabs API_KEY not set")
+
+testAPI = elevenLabsLogic.ElevenLabsAPI(apiKey=API_KEY)
+
+app = FastAPI()
+
+logger = logging.getLogger("generator")
+
+
+class CreateVoicePayload(BaseModel):
+    name: str
+    audio_file: UploadFile
+    description: str
+
+class SetVoicePayload(BaseModel):
+    selected_voice: str
+
+class ReadTextsPayload(BaseModel):
+    path_to_texts_file: str
+    path_to_save_files_dir: str
+    base_filename: str
+
+
+
+@app.post("/create_voice/")
+async def create_voice(payload: CreateVoicePayload):
+    try:
+        logger.info("Event grid event with payload {payload.audio_file.filename}")
+        uploaded_file_path = "{os.getcwd()}{os.sep}{payload.audio_file.filename}"
+        file_bytes = await payload.audio_file.read()
+        with open(uploaded_file_path, "wb") as f:
+            f.write(file_bytes)
+
+        testAPI.createOwnVoice(
+            name=payload.name,
+            pathToVoicesDir=uploaded_file_path,
+            description=payload.description
+        )
+
+        os.remove(uploaded_file_path)
+
+        return {"message": "Voice {payload.name} created successfully"}
```

Comment:
```suggestion
        return Response(status_code=status.HTTP_201_CREATED, content="Voice {payload.name} created successfully")
```
Dotyczy wszystkich pozostałych. Nie mniej jednak nie wszędzie będzie 201